### ISSUE TITLE
The ^M symbol appears at the end when using code snippets.

### ISSUE THREAD

**Issue by GJYAQXT** [NvChad/NvChad#1316]
**Describe the bug**
Is it a bug that code snippets appear with ^M symbols when writing Fortran programs using this project's configuration?

**Screenshots**
![MC`L~LI R55RRCO8F`ISQ4J](https://user-images.githubusercontent.com/63224494/177693371-ee41f7ef-626f-4855-ac90-d2f59e116a4b.png)

**Desktop (please complete the following information):**
 - Ubuntu (ssh)
 - Mobaxterm on Windows
 - neovim v7.2

**Comment by siduck** [comment]
![image](https://user-images.githubusercontent.com/59060246/177693793-b8691cbb-2d26-41a6-af77-671c0d3cc4fa.png)

i dont see any snippet issue with html  files

we didnt even configure this plugin, i think its an issue in the plugin itself

**Comment by siduck** [comment]
ok i get this issue with fortran only 

![image](https://user-images.githubusercontent.com/59060246/177694003-2f81b11e-8308-45c7-87ec-15e5d36d04fa.png)

**Comment by siduck** [comment]
Ok happens in lunarvim too

![image](https://user-images.githubusercontent.com/59060246/177708319-e0611f60-a91d-4ee6-b2eb-30c462e5a58b.png)

**Comment by GJYAQXT** [comment]
Yes, this problem is only found in Fortran syntax so far, do you have a solution?

**Comment by GJYAQXT** [comment]
When I removed the `\r` symbol in the fortran. json file and got the normal result, I thought it might be a problem that` \r\n` is a newline character on Windows, which appears as `^M `on Linux. When you replace it with` \n`, you get the correct newline character for Linux. It's really a question of `friendly-snippets` plugin.
